### PR TITLE
5947 : Update native desgin

### PR DESCRIPTION
**Pull request by wbenabdessalem**
Update The design of :
UINavigationBarStyle
Prevent the safe Area to be focused form the bottom

### CODE REVIEW

File: eXo/Sources/Controllers/HomePage/HomePageViewController.swift [hunk 1]
```diff
@@ -162,17 +167,14 @@ class HomePageViewController: eXoWebBaseController, WKNavigationDelegate, WKUIDe
...
-        // Page Register: [Domain]/portal/intranet/register
-        //(show navigation bar when the webview display this pages, because the pages don't contain a embedded navigation bar.
-        if (request.url?.absoluteString.range(of: serverDomain! + "/portal/login") != nil) || (request.url?.absoluteString.range(of: serverDomain! + "/portal/intranet/register") != nil) || (request.url?.absoluteString.range(of: serverDomain! + "/rest/state/status") != nil) {
-            self.navigationController?.setNavigationBarHidden(false, animated:true)
-        }else{
-            // Display the navigation bar at login or register page && disable the bar when login (register) is finished
-            // Home Page Address: portal/intranet/register (hide the navigation bar)
-            self.navigationController?.setNavigationBarHidden(true, animated:true)
+        // Display the navigation bar at login or register page && disable the bar when login (register) is finished
+        // Home Page Address: portal/intranet/register (hide the navigation bar)
```
Comment (ahamdi) [inline eXo/Sources/Controllers/HomePage/HomePageViewController.swift:174]:
```suggestion
        // Display the navigation bar at login or other pages accessible for anonymous users && display the bar when luser is logged in
        // Home Page Address: portal/dw 
```

Comment (wbenabdessalem) [reply eXo/Sources/Controllers/HomePage/HomePageViewController.swift:174]: Yes , is better and clear enough like that

Comment (ahamdi) [reply eXo/Sources/Controllers/HomePage/HomePageViewController.swift:174]: @wbenabdessalem could you commit this change ?

Comment (wbenabdessalem) [reply eXo/Sources/Controllers/HomePage/HomePageViewController.swift:174]: D'accord

Comment (hbenali) [reply eXo/Sources/Controllers/HomePage/HomePageViewController.swift:174]: Simply click here ;) 
image

File: eXo/Sources/Controllers/HomePage/HomePageViewController.swift [hunk 2]
```diff
@@ -51,6 +51,11 @@ class HomePageViewController: eXoWebBaseController, WKNavigationDelegate, WKUIDe
         self.navigationItem.title = NSLocalizedString("OnBoarding.Title.SignInToeXo", comment:"")
         self.navigationController?.isNavigationBarHidden = true
         setNavigationBarAppearance()
+        if UserDefaults.standard.bool(forKey: "isLoggedIn") {
```
Comment (ahamdi) [inline eXo/Sources/Controllers/HomePage/HomePageViewController.swift:54]:
```suggestion
         // check for logged in user
        if UserDefaults.standard.bool(forKey: "isLoggedIn") {
```

**Review by ahamdi — APPROVED** [review]

**Pull request merged**